qutip/tests/solve/test_stochastic_me.py

```diff
@@ -262,13 +262,18 @@ def test_general_stochastic():
     e_opsM = [spre(op) for op in e_ops]
 
     def d1(t, vec):
-        return L.mul_vec(t,vec)
+        return L.matmul_data(t, _data.Dense(vec)).to_array().ravel()
 
     def d2(t, vec):
         out = []
+        vec_d = _data.Dense(vec)
         for op in sc_opsM:
-            out.append(op.mul_vec(t,vec)-op.expect(t,vec)*vec)
-        return np.stack(out)
+
+            out.append(
+                _data.add(op.matmul_data(t,vec_d),
+                          vec_d, -op.expect_data(t,vec_d))
```

Comment:
It might be nice to write `scale=-op.expect_data(t,vec_d)` for those less familiar with the scale option to `.add`.